tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3520 @@
+              maxCommittableAge: 20
+              epochNearingThreshold: 30
+              congestionControlParameters:
+                minReferenceManaCost: '500'
+                increase: '500'
+                decrease: '500'
+                increaseThreshold: 800000
+                decreaseThreshold: 500000
+                schedulerRate: 100000
+                maxBufferSize: 3276800
+                maxValidationBufferSize: 100
+              versionSignalingParameters:
+                windowSize: 7
+                windowTargetRatio: 5
+                activationOffset: 7
+              rewardsParameters:
+                profitMarginExponent: 8
+                bootstrappinDuration: 1154
+                manaShareCoefficient: "2"
+                decayBalancingConstantExponent: 8
+                decayBalancingConstant: "1"
+                poolCoefficientExponent: 31
+              targetCommitteeSize: 32
+        baseToken:
+          name: "Shimmer"
+          tickerSymbol: "SMR"
+          unit: "SMR"
+          decimals: 6
+          subunit: "glow"
+        features:
+          - []
+
+    get-buildingBlock-response-example:
+      value:
+        strongParents:
+          - "0x4f746934d0075179a37ca3e807b7e793ec4ec9189ba31c9ad092d7497b01118c1000000000000000"
+          - "0x59daaaba1cd10cd268966376e17710d100c90826fe258753d461acac99b7e3341500000000000000"
+        weakParents:
+          - "0xebb058f1f1c8f1ac6508465deeee1dd090396f9e6e2b5c103eef5583f20a0c133a00000000000000"
+          - "0xb173e2d2b6f72600f00da84492ddb6eadee703dc39fb846725828059272093fb3b00000000000000"
+        shallowLikeParents:
+          - "0xdbf86c778d69f413296c2d4d3086d76c74bf4d719e6e95c03a3d9a955ed39bcf3e00000000000000"
+          - "0xf15b2ec698e396c9d2e69bc753246631aedebb657da09ba0e0c8d42518a1e8953e00000000000000"
+        latestFinalizedSlot: 1422
+        commitment:
+          protocolVersion: 3
+          slot: 1432
+          previousCommitmentId: "0xbf07dc16986166953f5a72c19d78642aab6cbde63964a2369ec3d425144eb42b3800000000000000"
+          rootsId: "0xa4dd36465af63d495d35a05f592d42a51511c153e1bae8fad00453c8cbb48727"
+          cumulativeWeight: '45678'
+          referenceManaCost: '12345'
+
+    post-blocks-response-example:
+      value:
+        blockId: "0xb173e2d2b6f72600f00da84492ddb6eadee703dc39fb846725828059272093fb3b00000000000000"
+
+    get-full-block-by-id-response-example:
+      value:
+        block: 
+          header:
+            protocolVersion: 3
+            networkId: "0"
+            issuingTime: "1692966346513784862"
+            slotCommitmentId: "0x8633b2eb1845fdecf12ee6c5e789c3cf1f0d0bbb3cee65cb5fb2757e995b5cd70000000000000000"
+            latestFinalizedSlot: 0
+            issuerId: "0x375358f92cc94750669598b0aaa55a6ff73310b90710e1fad524c0f911be0fea"
+          body:
+            type: 0
+            strongParents:
+              - "0x2de585533d9c9684747e49f70e5e620a4e6c3fd1c7d8b73520c198c7def29a050200000000000000"
```

Comment:
This is 80 chars instead of 72, probably the same in the whole file for every ID that contain a slot index